src/ActorMultiVertex.cpp

```diff
@@ -179,15 +179,24 @@ void ActorMultiVertex::SetNumVertices( std::size_t n )
 		AMV_start.vertices.resize( n );
 	}
 }
+ 
+void ActorMultiVertex::ResizeVertices(std::vector<RageSpriteVertex>& vertices, int size)
+{
+	if (vertices.capacity() < size)
```

Comment:
this emits a warning:
```
(...)/itgmania/src/ActorMultiVertex.cpp:185:33: warning: comparison of integer expressions of different signedness: ‘std::vector<RageSpriteVertex>::size_type’ {aka ‘long unsigned int’} and ‘int’ [-Wsign-compare]
  185 |         if (vertices.capacity() < size)
      |             ~~~~~~~~~~~~~~~~~~~~^~~~~~
```

Reply:
noted, will fix this in the future, the code has been well tested though so it shouldn't be an issue for now